packages/js/src/generators/library/library.ts

```diff
@@ -869,10 +869,11 @@ function determineEntryFields(
       };
     case 'rollup':
       return {
-        type: 'commonjs',
+        // Since we're publishing both formats, skip the type field.
+        // Bundlers or Node will determine the entry point to use.
         main: './index.cjs',
         module: './index.js',
-        // typings is missing for rollup currently
+        typings: './src/index.d.ts',
```

Comment:
We don't need to set `typings` since it should automatically match based on file name. It's only needed if the paths/names are different, e.g. `lib/index.js` vs `types/index.d.ts`. Also, if `generateExportsField` is specified, the types should match on those entries not `typings`.

Reply:
Perfect, I can tidy that up